registry.json

```diff
@@ -2954,5 +2954,14 @@
     "telegram": "StakeWithPride",
     "operatorName": "",
     "github": ""
-  }
+  },
+  {
+ "ticker": "SOLRP",
+ "poolId": "9728b10a926c048af938e5c52053319db5be921e8b698842c3afd3cc",
+ "social": {
+   "twitter": "SOLRP_StakePool"
+ },
+ "telegram": "solrp_ada_pool",
+ "github": "canad1an"
```

Comment:
Also please add in operatorName. This must be either your real first name, real full name or just blank.

Reply:
Done